### PR TITLE
Docs: slowfast 번역 및 PR 테스트

### PR DESCRIPTION
**Pull request by JDhyeok**
## 라이선스 동의
_변경해주시는 내용에 BSD 3항 라이선스가 적용됨을 동의해주셔야 합니다._<br />
_더 자세한 내용은 [기여하기 문서](https://github.com/PyTorchKorea/hub-kr/blob/master/CONTRIBUTING.md)를 참고해주세요._<br />
_동의하시면 아래 `[ ]`를 `[x]`로 만들어주세요._<br />

- [X] 기여하기 문서를 확인하였으며, 본 PR 내용에 BSD 3항 라이선스가 적용됨에 동의합니다.

## 관련 이슈 번호
_이 Pull Request와 관련있는 이슈 번호를 적어주세요._<br />
_이슈 또는 PR 번호 앞에 #을 붙이시면 제목을 바로 확인하실 수 있습니다. (예. #999 )_

- **이슈 번호**: #(숫자)

## PR 종류
_이 PR에 해당되는 종류 앞의 `[ ]`을 `[x]`로 변경해주세요._<br />
- [ ] 오탈자를 수정하거나 번역을 개선하는 기여
- [X] 번역되지 않은 모델 소개를 번역하는 기여
- [ ] 공식 허브의 내용을 반영하는 기여
- [ ] 위 종류에 포함되지 않는 기여

## PR 설명
_이 PR로 무엇이 달라지는지 대략적으로 알려주세요._

Slowfast 모델 번역 및 PR 연습용입니다.

### CODE REVIEW

File: facebookresearch_pytorchvideo_slowfast.md [hunk 1]
```diff
@@ -59,7 +60,8 @@ model = model.eval()
 model = model.to(device)
 ```
 
-Download the id to label mapping for the Kinetics 400 dataset on which the torch hub models were trained. This will be used to get the category label names from the predicted class ids.
+<!-- Download the id to label mapping for the Kinetics 400 dataset on which the torch hub models were trained. This will be used to get the category label names from the predicted class ids. -->
+토치 허브 모델 학습에 사용된 Kinetics 400 데이터셋 ID를 다운로드 합니다. ID는 라벨 매핑에 사용되며, 이는 예측 클래스 ID로부터 카테고리 라벨 이름을 가져오는데 사용됩니다.
```
Comment (HyunKyungHan) [inline facebookresearch_pytorchvideo_slowfast.md:64]: 첫 번째 문장을 **'토치 허브 모델 학습에 사용된 Kinetics 400 데이터셋에 레이블 매핑(label mapping)을 진행하기 위해 ID를 다운로드 합니다.'** 
이런 식으로 한 문장으로 번역하는 것은 어떤가요?

그리고 `label`을 `라벨`이라고 번역할지 `레이블`이라고 번역할지 정해서 [용어 사용 규칙 페이지](https://github.com/PyTorchKorea/tutorials-kr/blob/master/TRANSLATION_GUIDE.md)에 적용하면 좋을 것 같습니다!

Comment (HyunKyungHan) [inline facebookresearch_pytorchvideo_slowfast.md]: 첫 번째 문장을 **'토치 허브 모델 학습에 사용된 Kinetics 400 데이터셋에 레이블 매핑(label mapping)을 진행하기 위해 ID를 다운로드 합니다.'**
이런 식으로 한 문장으로 번역하는 것은 어떤가요?

그리고 `label`을 `라벨`이라고 번역할지 `레이블`이라고 번역할지 정해서 [용어 사용 규칙 페이지](https://github.com/PyTorchKorea/tutorials-kr/blob/master/TRANSLATION_GUIDE.md)에 적용하면 좋을 것 같습니다!

**Review by HyunKyungHan — CHANGES_REQUESTED** [review]

File: facebookresearch_pytorchvideo_slowfast.md (hunk 1 above)
Comment (JDhyeok) [reply facebookresearch_pytorchvideo_slowfast.md]: 문장 구조가 훨씬 자연스럽고 좋네요, 반영했습니다. 👍🏻 

좋은 의견 감사합니다. :)
